### PR TITLE
Add new module for vertx and routes

### PR DESCRIPTION
**Pull request by Sgitario**
This module covers some basic scenarios around reactive routes in general and also:
- Validation on request params, request body and responses.

### CODE REVIEW

**Comment by Sgitario** [comment]
@pjgg and @mjurc , this PR can be safely be ported to 1.11 branch if you consider so, just let me know.

File: 304-quarkus-vertx-routes/src/main/java/io/quarkus/qe/validation/annotations/UppercaseService.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package io.quarkus.qe.validation.annotations;
+
+import javax.enterprise.context.ApplicationScoped;
+
+@ApplicationScoped
+public class UppercaseService {
```
Comment (pjgg) [inline 304-quarkus-vertx-routes/src/main/java/io/quarkus/qe/validation/annotations/UppercaseService.java:6]: This class is not really necessary. You could move on these validations to `UppercaseValidator`

**Review by pjgg — CHANGES_REQUESTED** [review]

File: 304-quarkus-vertx-routes/src/main/java/io/quarkus/qe/validation/annotations/UppercaseService.java (hunk 1 above)
Comment (Sgitario) [reply 304-quarkus-vertx-routes/src/main/java/io/quarkus/qe/validation/annotations/UppercaseService.java:6]: I know it's not necessary, but I wanted to verify that we can inject services in our custom validators.

Comment (rsvoboda) [reply 304-quarkus-vertx-routes/src/main/java/io/quarkus/qe/validation/annotations/UppercaseService.java:6]: aka intentional, resolving and will merge

**Review by rsvoboda — APPROVED** [review]

**Pull request merged**